spec/factories.rb

```diff
@@ -18,6 +18,14 @@
       further_information { %w[true false].sample }
       further_information_details { Faker::Lorem.paragraph_by_chars(number: 300) }
 
+      phone_number { '07700 900 982' }
+      address_line1 { '42' }
+      address_line2 { 'Much Wow Street' }
+      address_line3 { 'London' }
+      address_line4 { 'England' }
+      country { 'GB' }
+      postcode { 'SW1P 3BT' }
+
```

Comment:
maybe should we use the faker to fill these details?

Reply:
I wanted to but I realised Faker provides US addresses e.g. postcode is zipcode and a different format. I guess I could use it for some of the fields but for now I'd prefer to run with this for now. I can address (😉) this a later PR when I add validation.

Reply:
Faker provides UK address if you configure it that way, I believe.

Reply:
Updated to use Faker now. Thanks for the link @benilovj!